src/components/InfiniteGrids/InfiniteVideoGrid.tsx

```diff
@@ -167,22 +167,24 @@ export const InfiniteVideoGrid: React.FC<InfiniteVideoGridProps> = ({
   // Right now we'll make the first request and then right after another one based on the resized columns
   return (
     <section className={className}>
-      {title && (
-        <GridHeadingContainer>
-          {!ready ? <StyledSkeletonLoader height={23} width={250} /> : <Title variant="h4">{title}</Title>}
-          {additionalLink && (
-            <Button
-              to={additionalLink.url}
-              size="medium"
-              variant="secondary"
-              iconPlacement="right"
-              icon={<SvgGlyphChevronRight width={12} height={12} />}
-            >
-              {additionalLink.name}
-            </Button>
-          )}
-        </GridHeadingContainer>
-      )}
+      <TitleWrapper>
+        {title && (
+          <GridHeadingContainer>
+            {!ready ? <StyledSkeletonLoader height={23} width={250} /> : <Title variant="h4">{title}</Title>}
+            {additionalLink && (
+              <AdditionalLink
+                to={additionalLink.url}
+                size="medium"
+                variant="secondary"
+                iconPlacement="right"
+                icon={<SvgGlyphChevronRight width={12} height={12} />}
```

Comment:
```suggestion
                icon={<SvgGlyphChevronRight  />}
```
it's 16px
![image](https://user-images.githubusercontent.com/51168865/128708820-f5987025-3bcf-4a57-b7ff-4e0d9db42806.png)